crmsh/ui_cluster.py

```diff
@@ -193,32 +193,22 @@ def looks_like_hostnames(lst):
     ssh         Create SSH keys for passwordless SSH between cluster nodes
     csync2      Configure csync2
     corosync    Configure corosync
-    storage     Partition shared storage (ocfs2 template only)
     sbd         Configure SBD (requires -s <dev>)
     cluster     Bring the cluster online
-    vgfs        Create volume group and filesystem (ocfs2 template only,
-                requires -o <dev>)
+    ocfs2       Configure OCFS2 (requires -o <dev>)
```

Comment:
Do mark it as Technical Preview.

Reply:
Mark which part as TP? The whole PR or just ocfs2 stage?

Reply:
I mean `crm cluster help init` will display something like:
```
*-o DEVICE, --ocfs2-device=DEVICE*
     .... NOTE: this is a Technical Preview
*ocfs2*
      Configure OCFS2 (requires -o <dev>) NOTE: this is a Technical Preview
```

Reply:
I think we don't need TP
just merge and use

Reply:
Changed
Thanks!